pom.xml

```diff
@@ -58,9 +58,9 @@
     <properties>
         <awaitility.version>3.0.0</awaitility.version>
         <farao.dependencies.version>1.13.0</farao.dependencies.version>
-        <farao.dichotomy.version>4.10.0</farao.dichotomy.version>
+        <farao.dichotomy.version>4.11.0-SNAPSHOT</farao.dichotomy.version>
         <gridcapa.starter.minio.version>1.0.0</gridcapa.starter.minio.version>
-        <gridcapa.rao.runner.version>1.13.0</gridcapa.rao.runner.version>
+        <gridcapa.rao.runner.version>1.14.0-SNAPSHOT</gridcapa.rao.runner.version>
```

Comment:
:information_source:  Waiting for gridcapa-rao-runner to be merged and released.